### PR TITLE
Siop verifier cleanup

### PR DESCRIPTION
**Pull request by sksadjad**
deleted/changed console logs with debugs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "build:clean": "lerna clean -y && lerna bootstrap && lerna run build:clean --concurrency 1 && yarn build:copyfiles && yarn build:schema-api",
     "build:schema-api": "lerna run extract-api && lerna run generate-plugin-schema",
     "build:copyfiles": "lerna run build:copyfiles",
+    "clean": "rimraf --glob **/dist **/coverage packages/**/node_modules node_modules packages/**/tsconfig.tsbuildinfo",
```
Comment (nklomp) [inline package.json:14]: There already is a build-clean doing roughly the same

File: packages/siopv2-oid4vp-rp-rest-api/src/SIOPv2RPRestAPI.ts [hunk 2]
```diff
@@ -241,14 +239,16 @@ export class SIOPv2RPRestAPI {
...
           .then((verifiedResponse: VerifiedAuthorizationResponse) => {
-            // console.log('verifiedResponse: ', JSON.stringify(verifiedResponse, null, 2))
-
             const wrappedPresentation = verifiedResponse?.oid4vpSubmission?.presentations[0]
             if (wrappedPresentation) {
               const credentialSubject = wrappedPresentation.presentation.verifiableCredential[0].credential.credentialSubject
-              console.log('AND WE ARE DONE!')
-              console.log(JSON.stringify(credentialSubject, null, 2))
-              console.log(JSON.stringify(wrappedPresentation.presentation, null, 2))
+              debug(
```
Comment (nklomp) [inline packages/siopv2-oid4vp-rp-rest-api/src/SIOPv2RPRestAPI.ts:245]: Please revert this one. This module is still being developed and tested. I will do a pass once it is ready

**Review by nklomp — CHANGES_REQUESTED** [review]
Requesting changes to block the merge. As the module still is seeing heavy development the most console.log lines are making sense.

In the mean time there are a few smaller comments like for instance the clean option you added, which can be addressed

File: package.json (hunk 1 above)
Comment (sksadjad) [reply package.json]: removed

**Comment by sksadjad** [comment]
I've added args for SIOPv2OID4VPRestClient constructor in this pr

**Pull request merged**